2024-04-05 12:04:25 LogWrapper init() ./logs/error.log
2024-04-05 12:04:31 Bot started
2024-04-05 12:05:06 CRASH: 'Series' object has no attribute 'LOSS'
